prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -384,9 +384,10 @@ def __list_mfa_devices__(self):
                 for page in list_mfa_devices_paginator.paginate(UserName=user.name):
                     for mfa_device in page["MFADevices"]:
                         mfa_serial_number = mfa_device["SerialNumber"]
-                        mfa_type = (
-                            mfa_device["SerialNumber"].split(":")[5].split("/")[0]
-                        )
+                        try:
+                            mfa_type = mfa_serial_number.split(":")[5].split("/")[0]
+                        except IndexError:
+                            mfa_type = "unknown"
```

Comment:
One last thing, could you review if this one are just the `hardware` ones? I'm approving the PR either way.

Reply:
Sure! I'm on this